test/lib/tasks/minify.integration.js

```diff
@@ -341,3 +340,67 @@ return;`;
 			`/resources/my/namespace/test.js (line 3, col 0, pos 48)`
 	}, `Threw with expected error message`);
 });
+
+test.serial("integration: minify with taskUtil and resources tagged with OmitFromBuildResult", async (t) => {
+	const {reader, writer, workspace} = createWorkspace();
+
+	const testFilePath1 = "/resources/my/namespace/test1.js";
+	const testFilePath2 = "/resources/my/namespace/test2.js";
+	const testFileContent1 = "function test(param1) { var variableA = param1; console.log(variableA); } test();";
+	const testFileContent2 = "function test(param2) { var variableB = param2; console.log(variableB); } test();";
+
+	const testResource1 = resourceFactory.createResource({
+		path: testFilePath1,
+		string: testFileContent1
+	});
+	const testResource2 = resourceFactory.createResource({
+		path: testFilePath2,
+		string: testFileContent2
+	});
+
+	await reader.write(testResource1);
+	await reader.write(testResource2);
+
+	const taskUtil = createTaskUtil(t);
```

Comment:
Let's rather create a mock like before and have the getTag() method always return `true` for this test. Something like this should hopefully work:

```js
{
	setTag: t.context.sinon.stub(),
	getTag: t.context.sinon.stub().returns(true),
	STANDARD_TAGS: {
		HasDebugVariant: "1️⃣",
		IsDebugVariant: "2️⃣",
		OmitFromBuildResult: "3️⃣"
	},
	registerCleanupTask: t.context.sinon.stub()
}
```

Then we can remove the `taskUtilHelper`.

My reasoning here is that `taskUtilHelper` is basically re-implementing the functionality of the real TaskUtil. I would like to avoid that in tests. IMHO we should test with _either_ use original _or_ a "static" mock of it.

Reply:
done